_posts/2023-10-16-OWASP-MAS-Importance-of-standards.md

```diff
@@ -0,0 +1,68 @@
+
+![The OWASP MAS Resources](/assets/images/posts/owasp-mas-importance-of-standards/owasp_mas_resources.png)
+
+### Comprehensive Coverage
+
+The OWASP MASVS standard is divided into various groups of security controls, representing critical areas of the mobile attack surface, including:
+
+- **MASVS-STORAGE**: Secure storage of sensitive data on a device (data-at-rest).
+- **MASVS-CRYPTO**: Cryptographic functionality used to protect sensitive data.
+- **MASVS-AUTH**: Authentication and authorization mechanisms used by the mobile app.
+- **MASVS-NETWORK**: Secure network communication between the mobile app and remote endpoints (data-in-transit).
+- **MASVS-PLATFORM**: Secure interaction with the underlying mobile platform and other installed apps.
+- **MASVS-CODE**: Security best practices for data processing and app maintenance.
+- **MASVS-RESILIENCE**: Resilience to reverse engineering and tampering attempts.
+
+### A Standard Backed-up by Standards
+
+The OWASP MASVS is intricately intertwined with various industry standards, underpinning its robustness and effectiveness. For instance, 
+
+- **MASVS-CRYPTO** relies on [NIST.SP.800-175B](https://csrc.nist.gov/pubs/sp/800/175/b/r1/final) and [NIST.SP.800-57](https://csrc.nist.gov/pubs/sp/800/57/pt1/r5/final), which provide established cryptographic guidelines and assurance, ensuring that sensitive data within mobile apps remains secure.
+- **MASVS-AUTH**, while the standard comprehensively covers app-side authentication and authorization, it recognizes the importance of validating security on the remote endpoint, referencing industry standards like the [OWASP Application Security Verification Standard (ASVS)](https://owasp.org/www-project-application-security-verification-standard/).
+- **MASVS-CODE** encourages developers to follow best practices from [OWASP Software Assurance Maturity Model (SAMM)](https://owasp.org/www-project-samm/) and [NIST.SP.800-218 Secure Software Development Framework (SSDF)](https://csrc.nist.gov/pubs/sp/800/218/final) to prevent vulnerabilities during development.
+- **MASVS-PRIVACY** draws inspiration from essential privacy regulations like GDPR, COPPA, CCPA, and ENISA, providing a foundation for privacy considerations.
+
+Additionally, the incorporation of [NIST OSCAL](https://pages.nist.gov/OSCAL/) further enhances the OWASP MASVS ecosystem. It standardizes the testing process, allowing the selection of specific standards and profiles for compliance, and providing a structured approach to express compliance evidence and testing results. This streamlines the testing methodology and facilitates transparency and understanding among all stakeholders. This integration not only ensures consistency but also future-proofs the standard's adaptability and relevance, making it an invaluable resource for mobile app security.
+
```

Comment:
```suggestion

## OWASP MAS Standards Industry Adoption

The OWASP MAS project has received significant industry support and recognition from a wide range of stakeholders. In particular, Google, as the provider of the Android mobile platform, has demonstrated its commitment to the project since 2021. Google is actively contributing feedback to the MASVS refactoring process through the App Defense Alliance (ADA) and its Mobile Application Security Assessment (MASA) program. By supporting the use of MASVS Level 1 requirements and promoting independent security assessments, Google is ensuring the security of its mobile app ecosystem.

Certification bodies such as CREST have also embraced MASVS by launching the OWASP Verification Standard (OVS) program, reinforcing the importance of OWASP standards in the area of application security. In addition, government institutions such as the European Payments Council, the Government of India, and others have adopted MASVS, underscoring its importance in ensuring secure mobile apps. Educational institutions and scientific research have also adopted MASVS, demonstrating its broad utility and impact across the industry. In addition, support from organizations such as the National Institute of Standards and Technology (NIST), the German Federal Office for Information Security (BSI), and ioXt, along with global standards initiatives, underscores the critical role MASVS plays in improving mobile app security on a global scale. Overall, the widespread adoption and endorsement of the OWASP MAS by these various entities highlights the importance of the project in shaping and maintaining high cybersecurity standards for mobile apps.

```